src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/openapi/ClientsApiController.java

```diff
@@ -113,18 +115,20 @@
     private final SubjectConverter subjectConverter;
     private final TokenCertificateConverter tokenCertificateConverter;
     private final OrphanRemovalService orphanRemovalService;
+    private final ServiceClientConverter serviceClientConverter;
 
     /**
      * ClientsApiController constructor
      */
+    @SuppressWarnings("checkstyle:ParameterNumber")
```

Comment:
Probably correct reaction to this, but parameter numbers warning is probably also an indication that this controller is getting too large. That's probably true, but there's not much we can do about the controller size & parameter numbers, since it is direct result of the API design.

Reply:
Yeah, I don't know if we can do anything (else than refactoring api) with this(?)